backend/routes.js

```diff
@@ -258,6 +267,21 @@ export default class Servers {
                         status = 201;
                     }
                     saveSetting('USERS', existingUsers);
+
+                    res.setCookie('plotly-auth-token', access_token, {'path': '/'});
+
+                    const db_connector_access_token = generateAndSaveAccessToken();
+                    res.setCookie('db-connector-auth-token', db_connector_access_token, {'maxAge': 300, 'path': '/'});
```

Comment:
same comments as in https://github.com/plotly/plotly-database-connector/pull/180/files#r141751696